lib/rototiller/task/params/command_flag.rb

```diff
@@ -0,0 +1,129 @@
+require 'rototiller/utilities/color_text'
+require 'rototiller/task/params/env_var'
+
+module Rototiller
+  module Task
+
+    class CommandFlag
```

Comment:
because it is not a part of the intent of this ticket.  future tickets are to move and fix add_option/switch